Sources/Core/API/TextureManager.swift

```diff
@@ -7,24 +7,47 @@
 import Foundation
 import CoreGraphics
 
+/**
+    This Enum decides how to handle Errors while loading images for a texture
+    - Note: Setting this only works in DEBUG mode
+ 
+    - Remark:
+        - **ignore**: Ignores the error
```

Comment:
If you put these as inline comments (using `///` on top of the actual cases) instead, then Xcode will show them as the user types a case, which is pretty neat. Like this:

```
/// Ignore any errors
case ignore
```

Reply:
This is super neat! Thanks for pointing out 💯 
Fixed 👍